.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST] One-liner description of feature request"
+labels: enhancement
```

Comment:
We have a "Feature Request" label. That should be used (and the title tag is unnecessary)

Reply:
Will fix. Title-wise it seemed like at least some issues used that convention, but I'm all for reducing redundancy between the title and labels.